mysql-test/lib/mtr_report.pm

```diff
@@ -126,7 +128,7 @@ sub _mtr_report_test_name ($) {
 
 sub mtr_report_test_skipped ($) {
   my ($tinfo)= @_;
-  $tinfo->{'result'}= 'MTR_RES_SKIPPED';
+  $tinfo->{'result'}= MTR_RES_SKIPPED;
```

Comment:
Each of this patch change where `'` is removed should be reverted if module is removed and I will not comment about that anymore.